applications/luci-app-https_dns_proxy/Makefile

```diff
@@ -10,7 +10,45 @@ LUCI_TITLE:=HTTPS DNS Proxy Web UI
 LUCI_DESCRIPTION:=Provides Web UI for HTTPS DNS Proxy
 LUCI_DEPENDS:=+luci-mod-admin-full +https_dns_proxy
 LUCI_PKGARCH:=all
-PKG_RELEASE:=5
+PKG_RELEASE:=7
+
+define Package/luci-app-https_dns_proxy/postinst
+#!/bin/sh
+# check if we are on real system
+if [ -z "$${IPKG_INSTROOT}" ]; then
+	uci -q batch <<-EOF >/dev/null
+	delete ucitrack.@https_dns_proxy[-1]
+	add ucitrack https_dns_proxy
+	set ucitrack.@https_dns_proxy[-1].init=https_dns_proxy
+	commit ucitrack
+EOF
+	if ! uci -q get dhcp.@dnsmasq[0].doh_backup_server >/dev/null; then
```

Comment:
What das doh_backup_server mean?

Reply:
It's the dnsmasq server setting before the package install. To be used on service stop and package uninstall. DoH is a short-hand for DNS-over-HTTPS.

Reply:
I don't know if that's okay if we add new uci option?